ballerina/client.bal

```diff
@@ -15,25 +15,43 @@
 // under the License.
 
 import ballerina/jballerina.java;
+import ballerina/uuid;
 import ballerina/http;
+import ballerina/websocket;
+
+const HTTP = "http://";
+const HTTPS = "https://";
+const WS = "ws://";
+const WSS = "wss://";
```

Comment:
Can't we define these without the `://` part? IMO If it's possible, better to define that way.